src/RegExpPattern.php

```diff
@@ -32,7 +35,7 @@ class RegExpPattern
     public static function tables()
     {
         $pattern = '/(?<creationScript>CREATE\s+TABLE\s+`(?<tableName>\S+)`\s+';
-        $pattern .= '\((?<tableDefinition>[^;\/]+)\)';
+        $pattern .= '\((?<tableDefinition>([^;\/]+?(.COMMENT.\'[^\']+((\'\')[^\']*)*\'(?!=\')))+.*?|[^;\/]+?)\)';
```

Comment:
Have done. I found however a new bug: This expression fails on a table comment, ie. the table is excluded from parsing